### DISCUSSION TITLE
Wrong deusizine description in Chinese(both Simplified and Traditional).

### DISCUSSION THREAD

**Discussion by YIMINNNNN** [FakeFishGames/Barotrauma#14568] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![批注 2024-09-04 162840](https://github.com/user-attachments/assets/ce9664db-7ed4-4004-9a0c-345ffea725cb)
![批注 2024-09-04 162033](https://github.com/user-attachments/assets/b9239017-e7fb-4d60-b959-566aced91392)

The descriptions texts of Deusizine and Saline in Chinese have been reversed ,I wrote annotations in the screenshots.
Red text was translated from the original text, and the blue text was copied from vanilla English file.
The same problem also occurs in traditional Chinese text.

### Reproduction steps

Changing the language to Chinese and you can find it.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! [This issue has reported previously](https://github.com/FakeFishGames/Barotrauma/discussions/14219) and we've fixed it in the unstable version, but the fix unfortunately hasn't made it to the public version yet. Should be fixed in the upcoming release though. Sorry for the inconvenience!
